projects/ion/src/lib/skeleton/skeleton.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/angular';
+
+import { IonSkeletonComponent } from './skeleton.component';
+
+const defaultProps = {
+  variant: 'rect',
+  width: 50,
+  height: 50,
+} as IonSkeletonComponent;
```

Comment:
```suggestion
const defaultProps: IonSkeletonProps = {
  variant: 'rect',
  width: 50,
  height: 50,
};
```
I think IonSkeletonProps is the correct type of these props. Avoid using `as type` when you can.